lib/translate/common.h

```diff
@@ -12,12 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "absl/strings/string_view.h"
+#include "lib/scsi_defs.h"
+
 namespace translator {
 
+constexpr absl::string_view kNvmeVendorIdentification = "NVMe    ";
+static void (*debug_callback)(const char *);
```

Comment:
This should not go in the header file, it should stay in the .cc where it was defined

Reply:
wouldn't that make it not be imported when `#include "common.h"`?
and since it is constant you must initialize it when you define it.\

Reply:
See https://github.com/googleinterns/scsi2nvme/pull/62#discussion_r445142974

Reply:
Also, it's not marked constant afaik

Reply:
oh i thought you were talking about the stringview